Docs improvements (#961)
anatawa12/AvatarOptimizer: .docs/content/docs/reference/remove-mesh-by-blendshape/index.md

In case polygons you want to remove are not removed, increase the `Tolerance` value.
In case polygons you do not want to remove are removed, decrease the `Tolerance` value.

> If you enabled `Automatically set BlendShape weight for preview when toggled`, the inspector will set BlendShape weight to 0 or 100 based on the toggle state. 

Reviewer: ```suggestion
If you enabled `Automatically set BlendShape weight for preview when toggled`, when you toggle specifying BlendShapes, their weight will be automatically set to 100 or 0.
```
